bigbluebutton-html5/imports/ui/components/breakout-room/hooks.ts

```diff
@@ -13,14 +13,21 @@ export const useStopMediaOnMainRoom = () => {
   const streams = useStreams();
 
   const stop = useCallback((presenter: boolean) => {
-    forceExitAudio();
-    VideoService.storeDeviceIds(streams);
-    stopVideo(exitVideo, streams);
-    logger.info({
-      logCode: 'breakoutroom_join_stop_media',
-      extraInfo: { logType: 'user_action' },
-    }, 'Joining breakout room closed audio in the main room');
-    if (presenter) finishScreenShare();
+    try {
+      forceExitAudio();
+      VideoService.storeDeviceIds(streams);
+      stopVideo(exitVideo, streams);
+      logger.info({
+        logCode: 'breakoutroom_join_stop_media',
+        extraInfo: { logType: 'user_action' },
+      }, 'Joining breakout room closed audio in the main room');
+      if (presenter) finishScreenShare();
+    } catch (error) {
+      logger.error({
+        logCode: 'breakoutroom_stop_media_error',
+        extraInfo: { error, logType: 'error' },
```

Comment:
```suggestion
        extraInfo: { errorMessage: error.message, errorStack: error.stack },
```
Raw errors are not sent to server.
Please send the desired fields directly - suggestion inline.